requirements.txt

```diff
@@ -4,7 +4,7 @@ html5lib>=1.1
 Pillow>=8.1.1
 pypdf>=3.1.0
 python-bidi>=0.4.2
-reportlab>=3.5.53
+reportlab=3.5.53
```

Comment:
reportlab>=3.5.53,<4.0

Reply:
check setup.py and tox.ini

Reply:
Made Changes on 
. setup.py (more important)
. tox.ini (for testing)
. requirements.txt